nightly-main/centos/7/Dockerfile

```diff
@@ -20,6 +22,8 @@ RUN yum install shadow-utils.x86_64 -y \
   sqlite \
   zlib-devel
 
+RUN ln -s /opt/rh/rh-git227/root/bin/git /usr/bin/git
```

Comment:
I dont like this, and happy to change to a better way to achieve this. the other option of course is adding to PATH but that also comes with its own set of issues

Reply:
@drexin @shahmishal wdyt?

Reply:
Let's provide a comment explaining why we need git 2.x in the Dockerfile.